spec/Section 5 -- Validation.md

```diff
@@ -1938,6 +1938,10 @@ A notable exception to typical variable type compatibility is allowing a
 variable definition with a nullable type to be provided to a non-null location
 as long as either that variable or that location provides a default value.
 
+In the example below, an optional variable is allowed to be used in an 
+non-null optional argument (`optionalBooleanArg`) because it provides a default 
```

Comment:
`it` is ambiguous (optional variable or non-null optional argument?) and further muddied with the naming `optionalBooleanArg` (how is an optional argument non-nullable?)

Proposal:
```
In the example below, an optional variable is allowed to be used in an 
non-null optional argument (`booleanArgWithDefault`) because the
field argument provides a default value in the schema.
```